extensions/Worker.Extensions.Shared/Configuration/ConfigurationExtensions.cs

```diff
@@ -1,7 +1,9 @@
-// Copyright (c) .NET Foundation. All rights reserved.
+﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using Microsoft.Extensions.Configuration;
+using System;
```

Comment:
nit: order imports (Microsoft. should come after System - we usually have System imports first, followed by everything else in alphabetically order)